modules/ROOT/pages/tools/neo4j-admin/validate-config.adoc

```diff
@@ -0,0 +1,62 @@
+:description: How to validate configurations using Neo4j Admin.
+[[neo4j-admin-validate-config]]
+= Validate configurations
+
+The `neo4j-admin server validate-config` command validates the Neo4j and Log4j configurations.
+It is located in the `bin` directory and is used to validate the configuration files before starting Neo4j.
+The command is available in Neo4j v5.5 and later.
+
+== Syntax
+
+The `neo4j-admin server validate-config` command has the following syntax:
+
+[source,role=noheader]
+----
+neo4j-admin server validate-config [-h] [--expand-commands] [--verbose]
+----
+
+== Options
+
+The `neo4j-admin server validate-config` command has the following options:
```

Comment:
Should the command be repeated every time for each section?

Reply:
Yes, because nobody is reading documentation as a book. Each section should be self-contained. So if a user is looking for the options of this command, they will not read the previous section.

Reply:
No, never mind then. I was just wondering :)